ExtensionBundle.Tests/DependencyValidationTests.cs

```diff
@@ -202,15 +178,26 @@ private class RuntimeFileComparer : IEqualityComparer<RuntimeFile>
         {
             public bool Equals([AllowNull] RuntimeFile x, [AllowNull] RuntimeFile y)
             {
-                return x.AssemblyVersion == y.AssemblyVersion &&
-                    x.FileVersion == y.FileVersion &&
-                    x.Path == y.Path;
+                if (x.AssemblyVersion is null && y.AssemblyVersion is null)
+                {
+                    return true;
+                }
```

Comment:
Adding this check as getting incorrect comparison result when AssemblyVersion is null

Reply:
The test should fail if Assembly version is null. That is what we are trying to do with TryParse below. If we have a known file that we build that causes this to fail, we can add an exception for that file.

Reply:
Updated